### DISCUSSION TITLE
Добавить в tableLogSelect опциональные параметры "tables" и "idCycle"

### DISCUSSION THREAD

**Discussion by simplepersonru** [totumonline/totum-ru-issues-and-discussions#411] [Есть идея, а что если...]
Хочу сделать кнопку "История" для каждого конкретного цикла (карточки). Чтобы было видна полная история по конкретной карточке, вне зависимости от того, какой пользователь их спровоцировал. 

Для этого сейчас нужно вызвать tableLogSelect, отфильтровать по нужной мне таблице и cycleIds через всякие циклы while и функциями, которые уменьшают списки итд, после этого передать в linkDataTable лог по пользователям. По сути писать фильтр для данных, но не с точки зрения SQL, а с точки зрения того как это делается в языках общего назначения. Как будто бы очень хочется видеть эти опциональные параметры в функции)

Соответственно тогда в таблицу Лог по пользователю нужно будет тоже добавить эти поля в header

**Comment by ttm-alexeypolunin** [comment]
Задача тысяч на 15 р. А если красиво заморочится - то на 150. Так-как такие невероятные деньги никто за нее не заплатит - будет в списке "может быть в сл жизни"

**Reply by simplepersonru** [reply]
За такую задачу готов расчехлить 5к рублей. Во столько я оцениваю свои трудозатраты по закостыливанию этого функционала своими силами в своем проекте. Понимаю, что это очень мало)
